_posts/2017/2017-09-12-five-compelling-reasons-why-you-should-care-about-terminology.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "5 compelling reasons why you should manage your terminology"
+date: "2017-09-12 09:30:00"
+image: blog-header/jellybears.jpg
+categories: tech-stories
+authors: ["Birgit"]
+---
+
+Communicating in a clear and consistent manner, and saving money at the same time.
+Sounds great?
+Let's tackle it!
+
+This one is about terminology work.
+And the importance to implement it in your company, your processes, your daily work.
+The **Who** and **How** and **Where** depend on every company's specific conditions and requirements.
+So better everyone figures that out for themselves.
+I'd like to lose a few words about **Why** terminology work matters:
+
+## Do you know what's meant?
+
+Terminology work has a very clear purpose: being consistent in your language and writing.
+Whether it is about developing a product, publishing press releases, or communicating within the company: talking about business issues in the same words, and using the same writing makes everyone's life easier and, primarily, more efficient.
+And productive.
+And satisfying.
+Everybody understands what is meant by using the same term, with a meaningful definition.
+It's as simple as that.
+
+## Go tell it on the mountain
+
+Once there is a consensus about the right terms to be used within a department, for a project, throughout the company, or whatever else: spread the word.
+Make an Excel sheet (did I really write that down?!), manage a list on your intranet, use one of these super professional terminology management tools (my absolute favorite!), or post messages on a notice board, but - please - communicate it.
+To everyone involved.
+And could be involved.
+And should be involved.
+And would like to be involved.
+Sounds like the whole company, right?
+
+Consistency amongst all departments saves everyone a lot of time and money, and, if you live it, demonstrates professionalism to your customers (see Customers in love).
```

Comment:
Maybe skip the professionalism at this point and only mention it in the customers section?